src/ui/public/visualize/visualize.html

```diff
@@ -6,7 +6,7 @@ <h2 aria-hidden="true"><i aria-hidden="true" class="fa fa-meh-o"></i></h2>
   </div>
   <div class="item bottom"></div>
 </div>
-<div div ng-hide="showNoResultsMessage()" class="vis-container">
+<div div ng-hide="showNoResultsMessage()" class="vis-container vis-container--legend-{{vis.params.legendPosition}}">
```

Comment:
I think this creates a tight coupling with the constants we use to set `legendPosition` value. Instead of doing this, let's instead call a helper method which returns the correct classes:

```
<div div ng-hide="showNoResultsMessage()" class="vis-container {{ getVisContainerClasses() }}">
```

See my comment here https://github.com/elastic/kibana/pull/7781#discussion_r72876812 for more details on this kind of pattern.